build.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Guarantee script exits with a non-zero return-code if a command below fails.
+set -e
```

Comment:
I would even recommend to go further
```suggestion
set -eu -o pipefail
```

since pipes are used

but in either case - requires review of the script to ensure that nothing ATM relies on tools exiting non-0 etc